roles/agnosticv/templates/governor.yml.j2

```diff
@@ -66,11 +66,8 @@ spec:
 {% endfor %}
 
   actions:
-{% for __action in anarchy_governor_standard_actions
- | union(_entry_points.keys() | list)
-%}
-{#   Skip action if entry point is explicitly set to null or empty string. #}
-{%   if __action not in _entry_points or _entry_points[__action] | default('', true) != '' %}
+{% for __action, __action_config in _actions.items() %}
+{%   if not __action_config.disabled | default(false) boolean %}
```

Comment:
syntax is weird?    did you mean `| bool?`

Reply:
Good catch, fixed!